lib/course_refresher.rb

```diff
@@ -129,12 +130,12 @@ def refresh_course(course)
         rescue StandardError, ScriptError # Some YAML parsers throw ScriptError on syntax errors
           @report.errors << $!.message + "\n" + $!.backtrace.join("\n")
           # Delete the new cache we were working on
-          FileUtils.rm_rf(@course.cache_path)
+          FileUtils.rm_rf(@course.cache_path)                     unless options[:no_directory_changes]
```

Comment:
Was this meant for all same kind of unless rows, or just for the quoted one? Now on the list above that it looks quite clear imo, but can take idents away of course if wanted.